client/test/e2e/arc/views/landing-view.js

```diff
@@ -17,6 +27,19 @@ var LandingView = (function () {
       }, 10000);
     };
 
+    this.openProcessManagerView = function() {
+      var self = this;
+      browser.driver.wait(function() {
```

Comment:
When I was cleaning up the tests I tried to make use of the ExpectedConditions API where possible. This would be a good spot for it:
```js
browser.driver.wait(EC.presenceOf(this.processManagerAppCommand), 10000);
```

See http://www.protractortest.org/#/api?view=ExpectedConditions.prototype.presenceOf and other tests that have `var EC = protractor.ExpectedConditions;` in them.